chemprop/cli/utils/conf.py

```diff
@@ -1,7 +1,3 @@
 from datetime import datetime
-import os
-from pathlib import Path
 
-CKPT_DIR = Path(os.getenv("CHEMPROP_CKPT_DIR", "model_checkpoints"))
-LOG_DIR = Path(os.getenv("CHEMPROP_LOG_DIR", "logs/chemprop"))
```

Comment:
why do you want to delete `LOG_DIR`? This allows us to group all the logs together under one directory if the user wants by not specifying a value for `--log-path` (or something to that effect).

Reply:
I think the logs should be saved next to the other Chemprop output by default so that when a user comes back to that output later, they can remember which log file goes with which output. I'll add another comment to this PR that shows a bit more of my thinking.